src/main/java/com/actiontech/dble/singleton/ThreadManager.java

```diff
@@ -0,0 +1,277 @@
+                DbleServer.getInstance().setTimerSchedulerExecutor(
+                        ExecutorUtil.createFixedScheduled(TIMER_SCHEDULER_WORKER_NAME, 2, ThreadChecker.getInstance()));
+                Scheduler.getInstance().init();
+                Iterator<PhysicalDbGroup> iterator = DbleServer.getInstance().getConfig().getDbGroups().values().iterator();
+                while (iterator.hasNext()) {
+                    PhysicalDbGroup dbGroup = iterator.next();
+                    LOGGER.info("dbGroup[{}] startHeartbeat...", dbGroup.getGroupName());
+                    dbGroup.startHeartbeat();
+                    LOGGER.info("dbGroup[{}] startDelayDetection...", dbGroup.getGroupName());
+                    dbGroup.startDelayDetection();
+                }
+                LOGGER.info("startXaIdCheckPeriod...");
+                XaCheckHandler.startXaIdCheckPeriod();
+                LOGGER.info("manual recover threadPool[{}] ... end ...", TIMER_SCHEDULER_WORKER_NAME);
+                break;
+            default:
+                throw new Exception("The recover operation of threadPool[" + threadName + "] is not supported");
+        }
+    }
+
+    public static void printAll() {
+        Thread[] threads = getAllThread();
+        StringBuilder sbf = new StringBuilder();
+        sbf.append("============== select all thread ============== start");
+        for (Thread thread : threads) {
+            sbf.append("\n \"");
+            sbf.append(thread.getName());
+            sbf.append("\" ");
+            sbf.append("#");
+            sbf.append(thread.getId());
+            sbf.append(", state: ");
+            sbf.append(thread.getState());
+            sbf.append(", stackTrace: ");
+            StackTraceElement[] st = thread.getStackTrace();
+            for (StackTraceElement e : st) {
+                sbf.append("\n\tat ");
+                sbf.append(e);
+            }
+        }
+        sbf.append("\n============== select all thread ============== end");
+        LOGGER.info(sbf.toString());
+    }
+
+    public static void printSingleThread(String threadName) throws Exception {
+        if (threadName != null && threadName.length() > 0) {
+            Thread[] threads = getAllThread();
+            StringBuilder sbf = null;
+            for (Thread thread : threads) {
+                if (thread.getName().equals(threadName)) {
+                    sbf = new StringBuilder();
+                    sbf.append("============== select thread[{" + threadName + "}] ============== start");
+                    sbf.append("\n \"");
+                    sbf.append(thread.getName());
+                    sbf.append("\" ");
+                    sbf.append("#");
+                    sbf.append(thread.getId());
+                    sbf.append(", state: ");
+                    sbf.append(thread.getState());
+                    sbf.append(", stackTrace: ");
+                    StackTraceElement[] st = thread.getStackTrace();
```

Comment:
need to pay attention to whether it causes STW?

Reply:
temporarily remove print requirement，subsequent creation of a separate issue